onebusaway-android/src/main/java/org/onebusaway/android/directions/realtime/RealtimeService.java

```diff
@@ -243,7 +243,7 @@ private void showNotification(ItineraryDescription description, int title, int m
                         PendingIntent.FLAG_CANCEL_CURRENT);
 
         NotificationCompat.Builder mBuilder =
-                new NotificationCompat.Builder(getApplicationContext())
+                new NotificationCompat.Builder(getApplicationContext(),Application.CHANNEL_REALTIME_ID)
```

Comment:
Could you please change `,Application.CHANNEL_REALTIME_ID` to `, Application.CHANNEL_REALTIME_ID` (add a space)?